### PR TITLE
clarified README and configuration-example

### PR DESCRIPTION
**Pull request by elv-seth**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,18 +3,12 @@
...
-  cd elv-client-js
-  npm install
-  node InitContentSpace.js <path-to-qfab-config> <path-to-content-fabric-dir> <private-key>
-  ```
-- Copy the generated TestConfiguration.json from ```elv-client-js``` to configuration.json in ```elv-core-js``` and ```elv-fabric-browser```
-
-  ```
-  cp TestConfiguration.json ../elv-core-js/configuration.json && cp TestConfiguration.json ../elv-fabric-browser/configuration.json
-  ```
+- Clone this repo along with ```elv-core-js```, and ```elv-client-js``` projects from github
```
Comment (elv-kevin) [inline README.md:6]: You don't need to clone elv-client-js, that's just included as a dependency.

File: configuration-example.js [hunk 2]
```diff
@@ -1,3 +1,12 @@
 const EluvioConfiguration = {
-  "config-url": "https://main.net955304.contentfabric.io/config"
-};
+  // Fabric Network
```
Comment (elv-kevin) [inline configuration-example.js:2]: this and the readme need details on
`fabricBrowserApps`,
`displayAppUrl`,
`embedAppUrl`

see https://core.v3.contentfabric.io/configuration.js for defaults

**Review by elv-kevin — CHANGES_REQUESTED** [review]

File: configuration-example.js (hunk 2 above)
Comment (elv-kevin) [reply configuration-example.js:2]: Please also specify that `coreUrl` should be changed to `http://localhost:8082` (local core url).

File: README.md [hunk 3]
```diff
@@ -3,18 +3,12 @@
...
+- Now set up a ```configuration.js``` file in the root directory using ```configuration-example.js``` as a guide. This can be done with 
+```
+cp configuration-example.js configuration.js
+```
+- Open the newly created ```configuration.js``` file and comment out all the config-url’s except for the one you wish to use (main, demo or test). Note that the config-url in elv-fabric-browser must match the config-url being used in elv-core-js otherwise the fabric-browser will not function.
```
[the remark this answers is not in the document]
Comment (elv-kevin) [reply README.md:11]: Maybe rephrase it as 'all instances of `config-url`'

File: README.md [hunk 4]
```diff
@@ -3,18 +3,12 @@
...
 
 - Run ```npm install && npm run serve``` in ```elv-core-js``` and ```elv-fabric-browser``` in separate terminal tabs
 - Open [http://localhost:8082](http://localhost:8082) in your browser
```
[the remark this answers is not in the document]
Comment (elv-seth) [reply README.md:14]: Including lines 13 and 14 where we specify the npm commands to run?

File: configuration-example.js [hunk 5]
```diff
@@ -1,3 +1,12 @@
...
+  "config-url": "https://test.net955203.contentfabric.io/config",
+  // "config-url": "https://main.net955305.contentfabric.io/config",
+  // "config-url": "https://demov3.net955210.contentfabric.io/config",
+
+  //v4
+  // "config-url": "https://host-468.contentfabric.io/config",
+
+  // Fabric Browser
```
[the remark this answers is not in the document]
Comment (elv-kevin) [reply configuration-example.js:10]: remove this comment (`// Fabric browser`)

File: configuration-example.js [hunk 6]
```diff
@@ -1,3 +1,27 @@
 const EluvioConfiguration = {
-  "config-url": "https://main.net955304.contentfabric.io/config"
-};
+  // Fabric Network
+  "config-url": "https://test.net955203.contentfabric.io/config",
```
Comment (elv-kevin) [inline configuration-example.js:3]: after looking at the code, fabric browser doesn't actually need config-url  so we can remove that from the config and references to it from the readme.

File: configuration-example.js [hunk 7]
```diff
@@ -1,3 +1,27 @@
...
+  // "config-url": "https://main.net955305.contentfabric.io/config",
+  // "config-url": "https://demov3.net955210.contentfabric.io/config",
+
+  //v4
+  // "config-url": "https://host-468.contentfabric.io/config",
+
+  // Fabric Browser
+  "coreUrl": "http://localhost:8082"
+
+  // manageAppUrl
```
Comment (elv-kevin) [inline configuration-example.js:13]: Uncomment these default app urls, they're necessary for the app to function.

**Review by elv-kevin — CHANGES_REQUESTED** [review]
Noted last couple of changes we need.

Please also add a note about modifying Core's config to add the local fabric browser to `apps`
